Makefile

```diff
@@ -275,3 +275,7 @@ e2e-exec: ginkgo
 
 e2e-destroy:
 	kind delete cluster --name capsule
+
+docs-lint:
+	cd docs/content && spellchecker *.md */*.md -d dictionary.txt
```

Comment:
Maybe at least use `npx spellchecker` instead?

Reply:
Also relying into the shell globbing is not ideal, simple quote the glob pattern to use the node implementation instead.

Reply:
Made suggested changes.
- Disable global install in workflow
- Execute `spellchecker-cli` with npx in Makefile without global install
- Pass files to tool with `-f` flag + quoted globs to bypass shell globbing and activate tool internal node globbing.